quora-api/src/main/java/com/upgrad/quora/api/exception/RestExceptionHandler.java

```diff
@@ -18,10 +20,29 @@
         );
     }
 
-    @ExceptionHandler(AuthenticationFailedException.class)
-    public ResponseEntity<ErrorResponse> AuthenticationFailedException(AuthenticationFailedException exe, WebRequest request) {
+    @ExceptionHandler(AuthorizationFailedException.class)
+    public ResponseEntity<ErrorResponse> authorizationFailedException(
```

Comment:
You should create a new authfailexception rather than amending the existing one as it will break the signin controller

Reply:
I didn't realize. I've overridden this .. Will make the change.